src/qaqc/download_and_flag_fxns/generate_summary_statistics.R

```diff
@@ -32,6 +33,9 @@ generate_summary_statistics <- function(site_param_df) {
       # Determine the slope of a point in relation to the point ahead and behind.
       slope_ahead = abs(front1 - mean)/15,
       slope_behind = abs(mean - back1)/15,
+      rollslope = roll_mean(slope_behind, n = 7, align = 'center', na.rm = F, fill = NA_real_),
+      # Add the standard deviation for points centered in a rolling slope of 7 points.
```

Comment:
question: does this mean that we are looking for the sd inside of a rolling 7 point period?

Reply:
yes! I'm exploring ways we might be able to determine sensor drift, especially instances where turbidity increases at a steady degree due to build-up. My idea is that we could identify instances where - if slope remains constant (give or take some threshold of difference) over a certain number of days, we flag it... still a mental work in progress, for sure!